Documentation/_exts/cilium_external_links.py

```diff
@@ -0,0 +1,20 @@
+from sphinx.writers.html import HTMLTranslator
+from sphinx.writers.html5 import HTML5Translator
+from sphinx.util.docutils import is_html5_writer_available
+
+# Makes all external links open in new tabs
```

Comment:
Usually we prefer infinitive verbs in comments

```suggestion
# Make all external links open in new tabs
```

Reply:
Noted. Thank you.